SalesforceCommerceCloud/pwa-kit
# Update createOrder to send SLAS USID

To complete the customer journey tracking handled by Einstein and Reports and Dashboards, eCOM sends a finishCheckout event when an order transitions from Created -> New status.

However, the finishCheckout event currently does not have any way to access the SLAS USID, which is used as the identifier to tag events as belonging to a particular user. Without this USID, finishCheckout events cannot be associated with other user events and this manifests as missing orders in Reports and Dashboards.

To provide eCOM with the USID, we will send the _sfdc_customer_id header when the order is created.
# Types of Changes




# Changes

- Add _sfdc_customer_id header to basket.createOrder

# How to Test-Drive This PR

- Create an order
- When the order is submitted, ensure that the _sfdc_customer_id header is included and that it contains the SLAS USID

## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:
